pkg/plugins/runtime/k8s/controllers/configmap_controller.go

```diff
@@ -42,6 +43,10 @@ func (r *ConfigMapReconciler) Reconcile(req kube_ctrl.Request) (kube_ctrl.Result
 	}
 
 	if err := r.VIPsAllocator.CreateOrUpdateVIPConfig(mesh); err != nil {
+		if kube_apierrs.IsConflict(err) {
```

Comment:
I believe this should be `store.IsResourceConflict`. We are translating Kube errors

Reply:
Yes, but ConfigStore didn't handle it and returned an original error. Fixed both ConfigStore and configmap_controller